modules/images/webp-uploads/settings.php

```diff
@@ -67,10 +67,19 @@ function webp_uploads_generate_webp_jpeg_setting_callback() {
 function webp_uploads_media_setting_style() {
 	?>
 	<style>
-	.form-table .perflab-generate-webp-and-jpeg th,
-	.form-table .perflab-generate-webp-and-jpeg td:not(.td-full) {
-		display: none;
-	}
+		<?php if ( ! is_multisite() ) : ?>
+		.form-table .perflab-generate-webp-and-jpeg th,
+		.form-table .perflab-generate-webp-and-jpeg td:not(.td-full) {
+			display: none;
+		}
+		<?php else : ?>
+		.form-table .perflab-generate-webp-and-jpeg > th {
+			font-size: 16px;
+		}
+		.form-table .perflab-generate-webp-and-jpeg > td:nth-child(2) {
+			display: none;
+		}
+		<?php endif; ?>
 	</style>
 	<?php
```

Comment:
Related to my above comment: Instead of all this CSS "hack", we should simply rely on the default markup (without the extra `td`) when in multisite. In other words, the only change to this function should be to return early (without rendering any CSS) when in multisite.